projects/storefrontstyles/scss/components/product/list/_facet-list.scss

```diff
@@ -1,35 +1,89 @@
-%cx-facet-list {
-  h4 {
-    padding: 0.4rem;
-  }
-  > .close {
-    display: none;
-  }
+body.modal-open {
+  overflow: hidden;
+}
 
-  @include media-breakpoint-up(lg) {
-    h4 {
-      display: none;
+%cx-dialog {
+  z-index: 11;
+  position: fixed; /* could also be absolute */
+
+  > .inner {
+    background-color: var(--cx-color-background);
+    transition: transform 0.3s ease-in-out;
+  }
+  &.active {
+    > .inner {
+      transform: translate3d(0, 0, 0);
     }
   }
+}
 
-  @include media-breakpoint-down(md) {
-    display: flex;
-    flex-direction: column;
-    position: fixed;
-    top: 0;
-    left: 0;
-    z-index: 11;
-    background-color: var(--cx-color-background);
-    height: 100%;
-    width: 100%;
+%cx-modal {
+  @extend %cx-dialog;
 
-    overflow: hidden;
-    transition: transform 0.3s ease-in-out;
+  top: 0;
+  left: 0;
+  height: 100%;
+  width: 100%;
 
+  transition: background-color 0.3s ease-in-out;
+  &.active {
+    background-color: rgba(0, 0, 0, 0.5);
+  }
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  > .inner {
     transform: translate3d(0, 100%, 0);
+  }
+}
 
-    &.active {
-      transform: translate3d(0, 0, 0);
+%cx-overlay {
```

Comment:
Is this used?

Reply:
This is causing the animation effect. Once the dialog becomes active, the transform kick in and neutralises to the zero position. The original position can be left, bottom, etc.